src/api/WebLink.js

```diff
@@ -28,6 +29,70 @@ class WebLink extends Item {
         const suffix: string = id ? `/${id}` : '';
         return `${this.getBaseApiUrl()}/web_links${suffix}`;
     }
+
+    /**
+     * Gets a Box weblink
+     *
+     * @param {string} id - Weblink id
+     * @param {(newItem: BoxItem) => void} successCallback - Function to call with results
+     * @param {Function} errorCallback - Function to call with errors
+     * @param {Array<String>} fields - Array of field strings
+     * @returns {Promise}
+     */
+    async getWeblink(
+        id: string,
+        successCallback: (newItem: BoxItem) => void,
+        errorCallback: Function,
+        { fields }: FetchOptions = {},
+    ): Promise<void> {
+        if (this.isDestroyed()) {
+            return;
+        }
+
+        const cache: APICache = this.getCache();
+        const key: string = this.getCacheKey(id);
+        this.errorCode = ERROR_CODE_FETCH_WEBLINK;
+        this.successCallback = successCallback;
+        this.errorCallback = errorCallback;
+
+        if (cache.has(key)) {
```

Comment:
I don't think this check is needed, as the `cache.get` return value is also checked in `findMissingProperties`.

Reply:
It is needed, because if it's `undefined`, `findMissingProperties` will return `[]` which length is `0`, then `successCallback` will execute.